sql/src/main/java/io/crate/analyze/AnalyzedCreateSnapshot.java

```diff
@@ -22,51 +22,60 @@
 
 package io.crate.analyze;
 
-import com.google.common.collect.ImmutableList;
-import org.elasticsearch.common.settings.Settings;
+import io.crate.expression.symbol.Symbol;
+import io.crate.sql.tree.Assignment;
+import io.crate.sql.tree.GenericProperties;
+import io.crate.sql.tree.Table;
 import org.elasticsearch.snapshots.Snapshot;
 
 import java.util.List;
+import java.util.function.Consumer;
 
-public class CreateSnapshotAnalyzedStatement implements DDLStatement {
+public class AnalyzedCreateSnapshot implements DDLStatement {
 
-    static final List<String> ALL_INDICES = ImmutableList.of("*", "-.blob_*");
+    public static final String[] ALL_INDICES = new String[]{"*", "-.blob_*"};
```

Comment:
Not sure if it's a good idea to make this public as string array, given that it could be mutated